node/src/command.rs

```diff
@@ -259,6 +259,7 @@ pub fn run() -> Result<()> {
 			let runner = cli.create_runner(&cli.run.normalize())?;
 
 			runner.run_node_until_exit(|config| async move {
+				let _ = &cli;
```

Comment:
@NukeManDan Do you have an idea why this is needed or where it came from?

Reply:
It has to do with [Disjoined Captures](https://doc.rust-lang.org/edition-guide/rust-2021/disjoint-capture-in-closures.html) in closures.

Now closures only captures the fields they need (in this case `cli.relaychain_args`) instead of the whole variable (`cli`). By adding `let _ = &cli` we get the old behaviour, in which the whole variable is captured. In our case I don't think it really matters, so we might as well leave the old behaviour